specs/core/0_beacon-chain.md

```diff
@@ -1288,22 +1298,22 @@ def get_matching_head_attestations(state: BeaconState, epoch: Epoch) -> List[Pen
 
 ```python
 def get_unslashed_attesting_indices(state: BeaconState,
-                                    attestations: List[PendingAttestation]) -> List[ValidatorIndex]:
+                                    attestations: Iterable[PendingAttestation]) -> Iterable[ValidatorIndex]:
```

Comment:
Is it possible to use `Set`?

I think we need to remember that we're writing this spec for humans' benefit first and foremost, and the mypy checker second, and `Iterable` is not really understandable to the average reader.

Reply:
There's `Sequence[<type>]` for `Union[List[<type>], Tuple[<type>, ...]] case:
```suggestion
                                    attestations: Iterable[PendingAttestation]) -> Sequence[ValidatorIndex]:
```

Might be more natural than `Iterable` and `Set`? 😄

Reply:
It's `Set`, since it's only used for iteration/inclusion.

Reply:
*non-order-specific iteration